superbench/cli/_handler.py

```diff
@@ -21,49 +21,108 @@ def check_argument_file(name, file):
         name (str): argument name.
         file (str): file path.
 
+    Returns:
+        str: Absolute file path if it exists.
+
     Raises:
         CLIError: If file does not exist.
     """
-    if file and not Path(file).exists():
-        raise CLIError('{} {} does not exist.'.format(name, file))
+    if file:
+        if not Path(file).exists():
+            raise CLIError('{} {} does not exist.'.format(name, file))
+        return str(Path(file).resolve())
+    return file
 
 
-def version_command_handler():
-    """Print the current SuperBench tool version.
+def split_docker_domain(name):
+    """Split Docker image name to domain and remainder part.
+
+    Ported from https://github.com/distribution/distribution/blob/v2.7.1/reference/normalize.go#L62-L76.
+
+    Args:
+        name (str): Docker image name.
 
     Returns:
-        str: current SuperBench tool version.
+        str: Docker registry domain.
+        str: Remainder part.
     """
-    return superbench.__version__
+    legacy_default_domain = 'index.docker.io'
+    default_domain = 'docker.io'
 
+    i = name.find('/')
+    domain, remainder = '', ''
+    if i == -1 or ('.' not in name[:i] and ':' not in name[:i] and name[:i] != 'localhost'):
+        domain, remainder = default_domain, name
+    else:
+        domain, remainder = name[:i], name[i + 1:]
+    if domain == legacy_default_domain:
+        domain = default_domain
+    if domain == default_domain and '/' not in remainder:
+        remainder = 'library/{}'.format(remainder)
+    return domain, remainder
 
-def deploy_command_handler(
-    docker_image,
+
+def process_config_arguments(config_file=None, config_override=None):
+    """Process configuration arguments.
+
+    Args:
+        config_file (str, optional): Path to SuperBench config file. Defaults to None.
+        config_override (str, optional): Extra arguments to override config_file,
+            following [Hydra syntax](https://hydra.cc/docs/advanced/override_grammar/basic). Defaults to None.
+
+    Returns:
+        DictConfig: SuperBench config object.
+        str: Dir for output.
+
+    Raises:
+        CLIError: If input arguments are invalid.
+    """
+    config_file = check_argument_file('config_file', config_file)
```

Comment:
Looks a little weird that one of the parameters is hard coded and just for logging.